### DISCUSSION TITLE
blog/the-journey-of-my-site

### DISCUSSION THREAD

**Discussion by giscus** [Parkerhiphop/my-site#14] [Announcements]
# blog/the-journey-of-my-site

關於「為什麼要架設個人網站？」、「為什麼要寫作？」、「如何選擇技術？」等大哉問

https://www.parkerchang.life/blog/the-journey-of-my-site

**Comment by 1orZero** [comment]
好讚的心得! 看完我也想來自己架個blog site！

**Comment by rongbinf** [comment]
從 Medium 而來，加油加油！
